projects/core/src/occ/adapters/checkout/converters/occ-replenishment-order-form-serializer.ts

```diff
@@ -27,10 +30,31 @@ export class OccReplenishmentOrderFormSerializer
   }
 
   /**
-   * Converts the date string to the Standard ISO 8601 format
+   * Converts the date string from a Standard ISO 860 format with or without time to valid OCC model format
+   * @Example
+   * Converts 2020-10-15T15:38:05 or 2020-10-15 2020-10-15T15:38:05-05:00
    */
   private convertDate(date: string): string {
-    const dateTime = '00:00:00';
-    return new Date(date).toISOString().split('T')[0] + 'T' + dateTime + 'Z';
+    const index = date.lastIndexOf(':');
+    let modelDate;
+    if (index > 0) {
+      modelDate = date.slice(0, index);
+    } else {
+      modelDate = `${date}T${this.getCurrentLocalTime()}`;
+    }
+
+    return this.dateTimePickerFormatterService.toModel(modelDate);
```

Comment:
I had to dig deeper and noticed the response that we get back from the backend returns a date 
- need to checkup with someone to checkout the backend as I believe we don't have to set the offset, and we only needed the datetime HH:mm 

Solution for this is just pass in HH:MM:00Z or HH:MM:00-00:00 (same thing)